src/contracts/pods/EigenPod.sol

```diff
@@ -706,6 +718,28 @@ contract EigenPod is IEigenPod, Initializable, ReentrancyGuardUpgradeable, Eigen
         return uint64(GENESIS_TIME + slot * SECONDS_PER_SLOT);
     }
 
+    /**
+     * @notice This function replicates the ssz hashing of a validator's pubkey, outlined below:
+     *  hh := ssz.NewHasher()
+     *  hh.PutBytes(validatorPubkey[:])
+     *  validatorPubkeyHash := hh.Hash()
+     *  hh.Reset()
+     */
+    function _hashPubkey(bytes memory validatorPubkey) internal pure returns (bytes32 pubkeyHash) {
+        require(validatorPubkey.length == 48, "Input should be 32 bytes in length");
```

Comment:
incorrect revert message, '32' in place of '48'.
might make more sense for this function to live in the BeaconChainProofs library, especially given it's 'pure' nature? IDK though, am not personally strongly opinionated on where this lives

Reply:
I agree, much more appropriate for BeaconChainProofs lib, moved